content/en/docs/architecture/jira.md

```diff
@@ -0,0 +1,91 @@
+lead to running into already fixed bugs which should never happen.
+
+If the `/cherrypick` command fails to run correctly due to conflicts, the Jira automation can still be used to assist in
+backporting. After manually creating the cherrypick PR, you can comment `/jira cherrypick OCPBUGS-XXX` to cherrypick a
+bug and link it to the PR. The automation will clone the provided bug and link the clone to the PR by adding it to the
+beginning of the PR's title. This command also supports cherrypicking multiple bugs (e.g.  `/jira cherrypick
+OCPBUGS-123,OCPBUGS-124`).
+
+## Integration with QE verification
+
+Normally, QE will verify that a fix correctly resolves a Jira bug once the bug transitions to the `ON_QA` state, which
+happens after the PR merges and the `release-controller` has assembled a new release payload containing the code.
+
+However, if a QE engineer is assigned to the Jira bug, they will also get notified when a pull request is created to fix
+the bug. The QE engineer can add the `qe-approved` label on the pull request to indicate that the fix has been verified
+before the PR even merges. In this case, when the nightly release containing the fix is accepted, the Jira bug will
+automatically transition from the `ON_QA` state to the `VERIFIED` state.
+
+## Configuration
+
+The configuration for the Jira integration can be found in the [openshift/release][0] repository.
+
+Sample config overriding the `target_version` for the `release-4.6` branch of the openshift/ceph-csi repository:
+{{< highlight yaml >}}
+  orgs:
+    openshift:
+      validate_by_default: true
+      # All bugs in all repos in the Openshift GitHub organization must have one of the following states, unless
+      # a repo or branchconfiguration for valid_states exists.
+      valid_states:
+      - status: NEW
+      - status: ASSIGNED
+      - status: ON_DEV
+      - status: POST
+      repos:
+        ceph-csi:
+          # If we reference a bug in openshift/ceph-csi and that bug has a dependent_bug, it must be in one of the following
+          # states.
+          dependent_bug_states:
+          - status: MODIFIED
+          - status: VERIFIED
+          branches:
+            release-4.8:
+              # Bugs for PRs that target the release-4.8 branch in the openshift/ceph-csi repo must have OCS 4.6.8
+              # as target_version.
+              target_version: OCS 4.6.8
+              validate_by_default: true
+{{< / highlight >}}
+
+The file defining all of the possible configuration options can be found [here][2].
+
+[0]: https://github.com/openshift/release/blob/master/core-services/jira-lifecycle-plugin/config.yaml
+[1]: https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/openshift_bugzilla_process
+[2]: https://github.com/openshift-eng/jira-lifecycle-plugin/blob/main/cmd/jira-lifecycle-plugin/config.go
+[3]: https://docs.google.com/document/d/1sxuq3f3dzhjt8mqYGL5_K4-2avhQwWC7gjEfZoABfEE/edit
```

Comment:
This document should be only accessible to Redhaters, right? It may be better to have this document as an .md file in the openshift-eng/jira-lifecycle-plugin repo.

Reply:
I don't think it matters that it's only accessible to Red Hatters. While the jira-lifecycle-plugin is obviously open-source, it's only intended for use by Red Hat and is very specific to our workflow. This isn't any different than having references to rover and source like we do in other parts of the docs site. It's also probably better to have a link to the Google Doc vs a yaml in the plugin's repo, as the Google Doc is more likely to be up-to-date when changes occur, especially if those changes don't result in any code changes on the plugin's side.